modules/social_features/social_group/social_group.install

```diff
@@ -613,3 +613,32 @@ function social_group_update_8905() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Make group creators as a group managers by default.
+ */
+function social_group_update_8906() {
+  $config_factory = \Drupal::configFactory();
+
+  $group_types = [
+    'public_group',
+    'open_group',
+    'closed_group',
+    'secret_group',
```

Comment:
@nechai I think it's better to clone this update hook and place relevant updates in 
`social_group_secret` and `social_group_flexible_group` modules.